packages/data-point/lib/reducer-list/factory.js

```diff
@@ -60,19 +73,16 @@ function parse (source) {
 module.exports.parse = parse
 
 /**
- * parses a raw transform
- * @param {string} source
+ * @param {Function} createTransform
+ * @param {Array} source
  * @return {Transform}
  */
-function create (source = []) {
+function create (createTransform, source = []) {
```

Comment:
- source should be first
- why are we moving this to injection instead of using it as a normal require?

Reply:
1. why should source be first?
2. passing the create function as a parameter is what we're already doing https://github.com/ViacomInc/data-point/blob/master/packages/data-point/lib/transform-expression/factory.js#L74